packages/notifi-react/lib/context/NotifiTargetContext.tsx

```diff
@@ -661,6 +668,151 @@ export const NotifiTargetContextProvider: FC<PropsWithChildren> = ({
     [],
   );
 
+  const getSignature = useCallback(
+    async (message: Uint8Array | string) => {
+      let signature: Uint8Array | string = '';
+
+      // TODO: Add logic for rest of the chains
+      switch (walletWithSignParams.walletBlockchain) {
+        case 'AVALANCHE':
+        case 'ETHEREUM':
+          signature = await walletWithSignParams.signMessage(
+            message as Uint8Array,
```

Comment:
we cast the message as `Uint8Array`, but we actually pass in `string` type, would it possibly cause issue.
Or we want to convert to actually Uint8Arry before passing in `signMessage`